news/2020-06-19-Bootable-jar-Wildfly-20.adoc

```diff
@@ -0,0 +1,107 @@
+= A bootable JAR for WildFly 20
+jfdenise
+2020-06-19
+:awestruct-tags: [wildfly, galleon, bootable, jar, bootable-jar]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+== Introduction
+
+Now that WildFly supports link:https://projects.eclipse.org/projects/technology.microprofile[Eclipse Microprofile] this seems the appropriate time to introduce 
+a way to package WildFly application as bootable JAR (bootable JAR packaging is well suited for microservices to be run on cloud and bare metal).
+
+As we know (bootable or fat JAR are very common), a bootable JAR allows us to package both the container and the application in a single JAR file and execute it with a simple 
+command such as ```java -jar myapp-wildfly.jar```.
+
+Packaging the server and your app in a JAR makes sense when you can control the set of server features you want to see present. 
+Packaging a complete WildFly server (250+MB) for a simple microservice would defeat the purpose of the bootable JAR. link:https://docs.wildfly.org/galleon/[Galleon] technology 
+server trimming capabilities has made bootable JAR for WildFly a valid solution.
+
+WildFly defines a set of link:https://docs.wildfly.org/20/Admin_Guide.html#defined-galleon-layers[Galleon layers] 
+(with new Galleon layers added nearly for each major WildFly releases) that you can combine in order to tailor the server to your application needs. 
+The tooling to build a bootable JAR relies on WildFly Galleon layers to provision the tailored server that is bundled in the JAR file.
+
+== WildFly JAR Maven plugin
+
+The project hosted in link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin[wildfly-extras/wildfly-jar-maven-plugin] defines a new Maven plugin allowing you
+to build a bootable JAR for WildFly (starting version 20.0.0.Final). Although currently incubating, the project has gained enough maturity that you can start using it. Obviously,
+things could still evolve in the near future. It is worth noting that the project has already entered the WildFly 
+feature process in order to become a fully qualified WildFly tooling. The new feature proposal can be found link:https://github.com/wildfly/wildfly-proposals/pull/306[there].
```

Comment:
s/there/here

Reply:
Why 'here' is better I don't know; it just is. ;)

Reply:
+1